application/src/Controller/Admin/UserController.php

```diff
@@ -126,18 +131,19 @@ public function changePasswordAction()
             $form->setData($this->params()->fromPost());
             if ($form->isValid()) {
                 $values = $form->getData();
+                if($currentUser && !$user->verifyPassword($values['current-password'])){
+                        $this->messenger()->addError('The current password entered was invalid.');
+                        return $view;
+                    }
                 $user->setPassword($values['password']);
                 $em->flush();
                 $this->messenger()->addSuccess('Password changed.');
-                return $this->redirect()->toRoute(null, array('action' => 'edit'), array(), true);
+                return $this->redirect()->toRoute(null, array('action' => 'edit'), array(), true);               
```

Comment:
Fix this part here that just adds invalid whitespace and you should be good to go.